composer.json

```diff
@@ -1,7 +1,7 @@
 {
     "name": "messagebird/php-rest-api",
     "description": "MessageBird REST API client for PHP",
-    "version": "1.4.1",
+    "version": "1.5.0",
```

Comment:
I think this field is unnecessary. @samwierema had problems until he removed this